com.unity.render-pipelines.universal/Documentation~/features/rendering-debugger.md

```diff
@@ -0,0 +1,162 @@
+# Rendering Debugger
+
+The **Rendering Debugger** window lets you visualize various lighting, rendering, and Material properties. The visualizations help you identify rendering issues and optimize Scenes and rendering configurations.
+
+This section contains the following topics:
+
+* [How to access the Rendering Debugger window](#how-to-access).
+
+    Information on how to access the **Rendering Debugger** window in the Editor, in the Play mode, and at runtime in Development builds.
+
+* [Rendering Debugger window sections](#ui-sections)
+
+    Descriptions of the elements and properties in the **Rendering Debugger** window.
+
+* [Navigation at runtime](#navigation-at-runtime)
+
+    How to navigate the **Rendering Debugger** interface at runtime.
+
+## <a name="how-to-access"></a>How to access the Rendering Debugger window
+
+The Rendering Debugger window is available in the following modes:
+
+* The Editor.
+
+* The Play mode.
+
+* At runtime in the standalone Unity Player, on any device. The window is only available in **Development builds**.
+
+When using the **Rendering Debugger** window in the Development build, clear the **Strip Debug Variants** check box in **Project Settings > Graphics > URP Global Settings**.
+
+Use one of the following options to open the **Rendering Debugger** window.
+
+**In the Editor**:
+
+* Select **Window > Analysis > Rendering Debugger**.
+
+* Press **Ctrl+Backspace** (**Ctrl+Delete** on macOS).
```

Comment:
In the editor, this doesn't work in Edit mode, only in Play mode, so I think this line should be removed.